python/lsst/ts/m2gui/config.py

```diff
@@ -0,0 +1,69 @@
+# This file is part of ts_m2gui.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["Config"]
+
+from dataclasses import dataclass
+
+
+@dataclass
+class Config:
```

Comment:
How do you plan to use it? I assume you will need to load (and load) those configurations from a file/CSC. If you plan to load from a file, then having a standard __init__ with an optional argument to load from a file would be much better. If this will be stored in CSC (as e.g. power_warning_motor is essentially CSC argument, and you don't expect GUI to run to trigger power_warning), much better approach is to store those in a CSC, provides CSC with possibility to load arguments, and reports those to GUI/EUI.

You can then easily spare report_config method, which will be replaced by a standard Error raised if class doesn't provide the argument.

Reply:
This is just a message/payload for `Model` to send to `TabConfigView`. The M2 cell code in cRIO will send the JSON packet to M2 GUI and CSC for the configuration details in it by the TCP/IP interface. The GUI or CSC will not load the cell configuration files actively. Thanks!